intro-HPC/ch_interactive_gent.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{HPC-UGent interactive and debug cluster}
+\label{ch:interactive_ugent}
+
+
+\section{Purpose}
+\label{sec:interactive_ugent_pupose}
+The purpose of this cluster is to give the user an environment where
+there should be no waiting in the queue to get access to a limited
+number of resources. This environment allows a user to immediatelty
+start working, and is the ideal place for interactive work such as
+development, debugging and light prodcution workloads (typically sufficient
+for training and/or courses). \\ This enviroment should be seen as an
+extension of the login nodes, instead of a dedicated compute resource.
+The interactive cluster is \emph{overcommitted}, which means that more CPU cores can be
+requested to jobs than physically exist in the cluster. Obviously, the performance of this cluster
+heavily depends on the workloads and the actual overcommit usage. Be aware that jobs can slow
+down or speed up during their execution.
+ \\
+Due to the restrictions and sharing of the CPU resources (see section~\ref{subsec:interactive_ugent_restrictions}) 
+jobs on slaking should normally start more or less immediately. As a consequence,
+typical workloads for this cluster should be limited to:
+\begin{itemize}
+  \item  Interactive jobs (see chapter~\ref{ch:running-interactive-jobs}) with lots of idle time
+         (courses, hands-ons, etc.) 
+  \item  Cluster desktop sessions (see chapter~\ref{ch:web_portal}) with lots of idle time
+         (courses, hands-ons, etc.)
+  \item  Small jobs for which performance is not an issue (courses, hands-ons, etc.)
```

Comment:
`Small jobs for which performance is not an issue (courses, hands-ons, etc.)` -> `Light jobs`? we should stress the "performance is not an issue" elsewhere

Reply:
ok